src/llmcompressor/pipelines/basic/pipeline.py

```diff
@@ -9,6 +9,7 @@
 from llmcompressor.modifiers.utils.pytorch_helpers import apply_pad_mask_to_batch
 from llmcompressor.pipelines.registry import CalibrationPipeline
 from llmcompressor.pytorch.utils.helpers import tensors_to_device
+from llmcompressor.utils.dev import dispatch_for_generation
```

Comment:
Is this also available in `llmcompresor.utils.dev`?

Reply:
Thanks for the catch, updated